### DISCUSSION TITLE
is it possible to have Node.js clients?

### DISCUSSION THREAD

**Discussion by rozek** [logux/logux#125] [Q&A]
The title says it already: is it possible to have a Node.js client, e.g., a small script that monitors a sensor and logs an action whenever that sensor's value changes? Or a client (running in a trusted environment) that persists the current state while the "servers" running "in the wild" are just "zero-knowledge" message brokers?

**Comment by ai** [comment]
Yes, it is possible. The [`Client`](https://logux.org/web-api/#client) class from `@logux/client`, I think, should work in modern Node.js environment.

**Reply by rozek** [reply]
good to know - I'll test it tomorrow.

Thanks for the info - and for the quick responses!
